regolith/helpers/u_milestonehelper.py

```diff
@@ -224,6 +224,13 @@ def db_updater(self):
                 if rc.finish:
                     rc.status = "finished"
                     doc.update({'end_date': now})
+                    notes = doc.get("notes", [])
+                    for i in range(len(notes)): 
```

Comment:
This is not pythonic at all. 
```
for note in notes:
```
is much more readable.   Then maybe use the string replace method, so something like
```
note.replace('()', '(x)')
```
Finally, modern python loves list comprehensions, so something like
```
notes_with_closed_items = [note.replace('()', '(x)') for note in notes]
```
something like that (I didn't test it worked, but something along those lines will work)

Reply:
Done